### DISCUSSION TITLE
Storybook

### DISCUSSION THREAD

**Discussion by jonsherrard** [criszz77/luna#84] [Ideas]
Hi!

Love the Luna setup. Was dying with the monorepo startup.

Would you consider adding Storybook? It's a real pain to setup and it'd be nice to see a solid "how to with tamagui" version.

**Comment by LunatiqueCoder** [comment]
Hey!

Thank you very much! I really appreciate it! 

I'm not sure if I would add it to the template since it adds maintenance burdens and I'm the only active maintainer, but I want to add different sections on how to integrate 3rd party services like i18next, Firebase (Auth&Firestore), and perhaps Redux. While definitely I'm intrigued about Storybook, I haven't tried integrating it into this setup yet and I can't promise that I'll find the time the near future. 🙈
